src/local-development/deploy-metadata.ts

```diff
@@ -44,17 +47,23 @@ export async function deployComponentMetadata(
 		return;
 	}
 
-	const componentUrl = getComponentApiUrl({ componentType, remoteComponentName, origin });
-
-	const axiosConfig: AxiosRequestConfig = {
-		headers: {
-			Authorization: 'Token ' + origin.apikey,
-		},
-		url: componentUrl,
-		method: 'PATCH',
-		data: metadataToUpdate,
-	};
-	await requestMakeApi(axiosConfig);
+	const checksumMatches = compareChecksumDeep(originChecksum, componentType, remoteComponentName, metadataToUpdate);
```

Comment:
Consider variable rename to `metadataChecksumMatches` or something with "metadata"